nomad/run-app_install-local.nomad.hcl

```diff
@@ -0,0 +1,35 @@
+job "app_install_scenario" {
+  type = "batch"
+  group "app_install" {
+    task "start_holochain" {
+      lifecycle {
+        hook = "prestart"
+        sidecar = true
+      }
+
+      driver = "raw_exec"
+      config {
+        command = "bash"
+        args = ["-c", "hc s clean && echo 1234 | hc s --piped create && echo 1234 | hc s --piped -f 8888 run"]
+      }
+    }
+
+    task "run_scenario" {
+      driver = "raw_exec"
+      env {
+        RUST_LOG = "info"
+      }
+      config {
+        command = abspath("result/bin/app_install")
```

Comment:
Will there be a way to have this reference a common path locally and remotely so that we can use the same scripts in dev as on a real environment?

Reply:
I'm hoping to allow this but I'm not yet sure what is the best way. I think at the very least it should be possible to provide either a path or a URL and if it's a URL then the binary gets downloaded as an artefact before being run.